app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -44,12 +45,7 @@ import org.mozilla.fenix.R
 import org.mozilla.fenix.components.FenixSnackbar
 import org.mozilla.fenix.components.StoreProvider
 import org.mozilla.fenix.components.metrics.Event
-import org.mozilla.fenix.ext.bookmarkStorage
```

Comment:
One more nit and then we're there! Our lint checks don't allow wildcard imports so these all need to be declared separately. The IDE probably auto changed this to try to be smart :/ but you can change the Android Studio settings for this in Preferences -> Editor -> Code style -> Kotlin and then select "Use single name import" 👍

Reply:
Single name imports it is! 😊